tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -220,6 +220,37 @@
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.GetAzureRmEventHubAuthorizationRule","Get-AzureRmEventHubAuthorizationRule","0","2090","The ValidateNotNullOrEmpty attribute has been added to parameter 'AuthorizationRuleName' for cmdlet 'Get-AzureRmEventHubAuthorizationRule'.","Remove the ValidateNotNullOrEmpty attribute from parameter 'AuthorizationRuleName'."
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.GetAzureRmEventHubAuthorizationRule","Get-AzureRmEventHubAuthorizationRule","0","2100","The parameter 'NamespaceName' in cmdlet 'Get-AzureRmEventHubAuthorizationRule' is no longer in the parameter set '__AllParameterSets'.","Add parameter 'NamespaceName' back to the parameter set '__AllParameterSets'."
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.GetAzureRmEventHubAuthorizationRule","Get-AzureRmEventHubAuthorizationRule","0","2100","The parameter 'EventHubName' in cmdlet 'Get-AzureRmEventHubAuthorizationRule' is no longer in the parameter set '__AllParameterSets'.","Add parameter 'EventHubName' back to the parameter set '__AllParameterSets'."
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.GetAzureEventHubKey","Get-AzureRmEventHubKey","0","2100","The parameter 'NamespaceName' in cmdlet 'Get-AzureRmEventHubKey' is no longer in the parameter set '__AllParameterSets'.","Add parameter 'NamespaceName' back to the parameter set '__AllParameterSets'."
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.GetAzureEventHubKey","Get-AzureRmEventHubKey","0","2100","The parameter 'EventHubName' in cmdlet 'Get-AzureRmEventHubKey' is no longer in the parameter set '__AllParameterSets'.","Add parameter 'EventHubName' back to the parameter set '__AllParameterSets'."
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.NewAzureRmEventHub","New-AzureRmEventHub","0","2060","The position of parameter 'EventHubName' has changed for parameter set '__AllParameterSets' for cmdlet 'New-AzureRmEventHub'.","Revert the position change made to parameter 'EventHubName' for parameter set '__AllParameterSets'."
```

Comment:
New-AzureRmEventHub  EventHubName position has changed.  This is a breaking change.  Please change it back